test/e2e/csi/e2e_suite_test.go

```diff
@@ -74,10 +74,10 @@ func handleFlags() {
 }
 
 func execTestCmd(cmds []testCmd) {
-	err := os.Chdir("../..")
+	err := os.Chdir("../../..")
```

Comment:
Just a side note. Working directory is akin to global variables (i.e. global mutable state). Which are generally best to be avoided. More so for relative modification if it's value. It breaks often and it's hard to work with (e.g. there's no way to understand what exactly this does unless one already has current knowledge of the rest of the system). I'm not suggesting any particular change just voicing that `Chdir`, `cd` in shell etc... is best to be avoided if at all possible (or it's change be limited to a small section of code) and it's almost always possible (`dirname "$0` or go's alternatives work well enough).

Reply:
This is a port of 3rd party tests, so I do not want to make unnecessary changes.

Reply:
I agree, I just needed to get it out of my system :-)